src/NzbDrone.Core/Datastore/Migration/001_initial_setup.cs

```diff
@@ -50,6 +50,7 @@ protected override void MainDbUpgrade()
                 .WithColumn("Status").AsInt32()
                 .WithColumn("Overview").AsString().Nullable()
                 .WithColumn("Images").AsString()
+                .WithColumn("FlatFileName").AsString()
```

Comment:
This needs to be added in a new migration to avoid breaking all existing installs